swift/ZKSyncSDK/ZKSyncSDK/ZKSyncSDK+Types.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ZksCryptoSecretKey.swift
+//  ZKSyncSDK-UI
+//
+//  Made with ❤️ by Matter Labs on 10/23/20
+//
+
+import Foundation
+
+public class ZKSecretKey {
+    private var content: Data
+    
+    init(_ content: Data) {
+        self.content = content
```

Comment:
I think it would be better if there will be an assertion to check that content length is the same as expected one for the designated type. Not sure how to express it in terms of `swift` though.